src/pubtools/_quay/push_docker.py

```diff
@@ -557,10 +559,13 @@ def sign_new_manifests(self, docker_push_items: List[Any]) -> List[Tuple[str, st
                             + ":"
                             + tag
                         )
+                        registry = self.dest_registries[0]
+                        pub_reference = f"{registry}/{repo}"
```

Comment:
> The critical.identity.docker-reference field should follow the same rules / design as for the GPG signatures:
> For every tag the image is published under, there should be a signature with identity set to exactly that tag.

Does tag need to be included in identity?

Reply:
Cosign doesn't include it in docker-reference. I guess it's because tag is option when you sign with cosign